### PR TITLE
Camera doc updates

### PR DESCRIPTION
**Pull request by davidplowman**
Some general fixups, but most of it Pi 5 related.

@naushir Do you want to check the 2nd commit accurately describes the hdr option? (Or at least, the hdr option once you've sorted it out...)

The final commit addresses https://github.com/raspberrypi/documentation/issues/3235.

### CODE REVIEW

**Review by aallan — APPROVED** [review]

**Comment by aallan** [comment]
Looks good to me. @naushir ?

File: documentation/asciidoc/computers/camera/libcamera_apps_building.adoc [hunk 1]
```diff
@@ -82,17 +79,15 @@ cd libcamera
...
 ----
-meson build --buildtype=release -Dpipelines=raspberrypi -Dipas=raspberrypi -Dv4l2=true -Dgstreamer=enabled -Dtest=false -Dlc-compliance=disabled -Dcam=disabled -Dqcam=disabled -Ddocumentation=disabled -Dpycamera=enabled
+meson build --buildtype=release -Dpipelines=rpi/vc4,rpi/pisp -Dipas=rpi/vc4,rpi/pisp -Dv4l2=true -Dgstreamer=enabled -Dtest=false -Dlc-compliance=disabled -Dcam=disabled -Dqcam=disabled -Ddocumentation=disabled -Dpycamera=enabled
 ----
 
 Users of Raspberry Pi OS can instead use
 
 ----
-meson build --buildtype=release -Dpipelines=raspberrypi -Dipas=raspberrypi -Dv4l2=true -Dgstreamer=enabled -Dtest=false -Dlc-compliance=disabled -Dcam=disabled -Dqcam=enabled -Ddocumentation=disabled -Dpycamera=enabled
+meson build --buildtype=release -Dpipelines=rpi/vc4,rpi/pisp -Dipas=rpi/vc4,rpi/pisp -Dv4l2=true -Dgstreamer=enabled -Dtest=false -Dlc-compliance=disabled -Dcam=disabled -Dqcam=enabled -Ddocumentation=disabled -Dpycamera=enabled
```
Comment (naushir) [inline documentation/asciidoc/computers/camera/libcamera_apps_building.adoc:88]: The lite/full fat commands seem to be the same now.  Should we remove the duplicate?

Also one minor nit - should we start using ``meson setup build ...`` now?

**Comment by naushir** [comment]
Small Q above, but otherwise looks good to me.

**Comment by davidplowman** [comment]
Yes to both, though I think qcam might be different in both commands? I'm inclined just to drop it.

I agree, "meson setup" is better, IIRC it whinges (though works) if you don't use it. Let me re-push this PR.

**Review by aallan — APPROVED** [review]

**Pull request merged**